src/main/flight/position.c

```diff
@@ -196,9 +228,7 @@ void calculateEstimatedAltitude(void)
     zeroedAltitudeDerivative = pt2FilterApply(&altitudeDerivativeLpf, zeroedAltitudeDerivative);
 
 #ifdef USE_VARIO
-    estimatedVario = lrintf(zeroedAltitudeDerivative);
-    estimatedVario = applyDeadband(estimatedVario, 10); // ignore climb rates less than 0.1 m/s
-    estimatedVario = constrain(estimatedVario, -1500, 1500);
+    estimatedVario = calculateEstimatedVario(baroAltCm, dTime);
```

Comment:
GPS data is now excluded from having vario signals. Is this by accident?

Reply:
@KarateBrot - This PR was trying to back track some changes trying to identify why Vario value is not being updated.
At this point it still does not work so this PR does not make any sense at this point.

Maybe this is caused by the filtering change on baro?